config/routes.rb

```diff
@@ -136,6 +137,8 @@
 
       # api_v1_ping_times GET /api/v1/ping_times
       get 'ping-times/:network', to: 'api#ping_times', as: 'ping_times'
+
+      post 'ping-thing/:network/:token', to: 'ping_thing#create', as: 'ping_thing'
```

Comment:
```suggestion
      post 'ping-things/:network/:token', to: 'ping_things#create', as: 'ping_things'
```
I'd also move params to body from url since it's a POST request. What is the purpose of `:token` param here in link?